dashboard/data/utils.py

```diff
@@ -6,7 +6,7 @@ def generate_dummy_links_dataframe(compound_ids: list[str]) -> pd.DataFrame:
         f"[EOS{i+1}](https://ecbd.eu/compound/EOS{i+1})"
         for i, _ in enumerate(compound_ids)
     ]
-    return pd.DataFrame({"CMPD ID": compound_ids, "EOS": eos}).set_index("CMPD ID")
+    return pd.DataFrame({"EOS": compound_ids, "EOS": eos}).set_index("EOS")
```

Comment:
Should this return a DF with duplicated `EOS` column?

Perhaps this function is not needed at all - we might generate the link dynamically based on the raw EOS number and link template

Reply:
my bad just mindlessly replaced all `CMPD ID` with `EOS`